.github/workflows/lint.yml

```
@@ -0,0 +1,27 @@
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run ESLint
+        run: npm run lint
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)